modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -342,12 +342,46 @@ definition {
 	}
 
 	@description = "Verify that the user can order the results inside Assigned to My Roles by Last Activity Date"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOrderAssignedtoMyRolesTasksByLastActivityDate {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		Navigator.openURL();
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry 1");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry 2");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Notifications.goToFilterValues(filterValues = "Last Activity Date");
 
-		// TODO LRQA-68839 CanOrderAssignedtoMyRolesTasksByLastActivityDate pending implementation
+		AssertTextEquals.assertPartialText(
+			key_workflowAssetTitle = "Blogs Entry 1",
+			key_workflowTask = "Review",
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_LAST_ACTIVITY_DATE",
+			value1 = "9 Seconds Ago");
 
+		AssertTextEquals.assertPartialText(
+			key_workflowAssetTitle = "Blogs Entry 2",
+			key_workflowTask = "Review",
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_LAST_ACTIVITY_DATE",
+			value1 = "8 Seconds Ago");
 	}
```

Comment:
This parameter/path (ASSIGNED_TO_MY_ROLES_TABLE_LAST_ACTIVITY_DATE) you used is variable, and will fail if run repeatedly, ideally, use a path that looks for the index of the item in the table, to ensure that the position is being respected according to the filter.
Could create this path:

MY_WORKFLOW_TASK_ITEM_POSITION //tr[@data-qa-id='row'][${key_rowPosition}]//a[contains(text(),'${key_title}')]
After adding it to some .path file, you could also create a Macro like : viewMyWorkflowTaskItemPosition

Since you are going to use this assert function more than once!

Same thing that i have noted at this [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/844), when making changes to both codes, please ensure they were made in the same place so there are no conflicts.

Thank you!